pkgs/by-name/di/dim/package.nix

```
@@ -0,0 +1,100 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  buildNpmPackage,
+  makeWrapper,
+  ffmpeg_5,
+  git,
+  pkg-config,
+  sqlite,
+  libvaSupport ? stdenv.hostPlatform.isLinux,
+  libva,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "dim";
+  version = "unstable-2023-12-29";
+  src = fetchFromGitHub {
+    owner = "Dusk-Labs";
+    repo = pname;
```

```suggestion
    repo = "dim";
```

https://github.com/NixOS/nixpkgs/issues/277994